app/controllers/api/teams_controller.rb

```diff
@@ -15,16 +15,14 @@ def self.policy_class
 
   # GET /api/teams
```

Comment:
update #create !
override build_entry to set current_user.
remove create method completely

I added authorize methods in crud_controller. maybe this is going to break some other controllers.
please check those too. 
we got now DRY_CRUD, so use it :)